.github/scripts/defaults.sh

```diff
@@ -67,6 +67,7 @@ mapping["chipyard-shuttle"]=" CONFIG=ShuttleConfig"
 mapping["chipyard-multiclock-rocket"]=" CONFIG=MulticlockRocketConfig"
 mapping["chipyard-nomem-scratchpad"]=" CONFIG=MMIOScratchpadOnlyRocketConfig"
 mapping["chipyard-constellation"]=" CONFIG=SharedNoCConfig"
+mapping["chipyard-aes256ecb"]=" CONFIG=AES256ECBRocketConfig verilog"
```

Comment:
Do verilator builds of calliptra not work

Reply:
It should, I just haven't tested it yet. I'll add this in later on.

Reply:
Ok it compiles in Verilator now (doesn't run but that can be addressed later).